thesis-de/content-de.tex

```diff
@@ -182,6 +197,112 @@ \subsection{Datenbanksysteme}
   \1 hadoop
   \1 hive
 \end{outline}
+\nl%
+
+\subsubsection{Vorbereitung der Evalution}
+\label{subsubsec:vorbereitung_der_evalution}
+Als Vorbereitung für die Evalution der Datenhaltungssysteme wurden
+mehrere Kriterien definiert, welche uns einen Leitpfaden für die 
```

Comment:
du ändert die verbform hier, bleib bei passiv.
"...wurden mehrere Kriterien definiert, welche einen Leitpfaden für die  spätere Arbeit geben."
Konjunktiv soll vermieden werden sofern er nicht notwendig ist (in dem fall 'sollen').